test/parallel/parallel.status

```diff
@@ -5,17 +5,36 @@ prefix parallel
 # sample-test                       : PASS,FLAKY
 
 [true] # This section applies to all platforms
+test-child-process-fork-net2      : PASS,FLAKY
+test-tick-processor               : PASS,FLAKY
```

Comment:
https://github.com/nodejs/node/pull/2376#issuecomment-137402700 failed on windows too, more than once. If you feel strongly about it, I can put it under linux and windows only.